frontend/src/components/NavBar/styles.js

```diff
@@ -33,6 +33,33 @@ const styles = theme => ({
   h5: {
     marginBottom: theme.spacing(2),
   },
+  container: {
+    width: '90%',
+    display: 'flex',
+    flexDirection: 'column-reverse',
+    height: '100%',
+    borderCollapse: 'collapse',
+    paddingLeft: '2px',
```

Comment:
I would make `paddingLeft` with the value `0px`. This would center the buttons instead of having them slightly more aligned to the right than normal

Reply:
Added!